pythonwhat/checks/check_object.py

```diff
@@ -354,7 +354,7 @@ def check_keys(state, key, missing_msg=None, expand_msg=None):
 
     def get_part(name, key, highlight):
         if isinstance(key, str):
-            slice_val = ast.Str(s=key)
+            slice_val = ast.Constant(value=key)
```

Comment:
Is this a change in the `ast` library in `python3.12`?

Reply:
It's 2 different deprecations:

```
DeprecationWarning: ast.Str is deprecated and will be removed in Python 3.14; use ast.Constant instead
    slice_val = ast.Str(s=key)
```
```
DeprecationWarning: Attribute s is deprecated and will be removed in Python 3.14; use value instead
    return Constant(*args, **kwargs)
```

Not strictly mandatory for this work, but at least it's less work for next time (and fewer deprecation warnings making noise).